src/main/webapp/courseLoader.js

```diff
@@ -475,6 +484,10 @@
     postRatingProperties(urlAndData[0], urlAndData[1]);
   }
 
+  document
+    .getElementById("myBtn")
+    .addEventListener("click", passRatingProperties);
```

Comment:
Probably cleaner to have an onclick attribute on the html

Reply:
We had an onclick but for some reason it would not work so Laura decided to move to this implementation